RecoLocalTracker/SubCollectionProducers/src/TrackClusterRemover.cc

```diff
@@ -192,7 +191,7 @@ namespace {
         auto const & thit = reinterpret_cast<BaseTrackerRecHit const&>(hit);
         auto const & cluster = thit.firstClusterRef();
 	if (cluster.isStrip()) collectedStrips[cluster.key()]=true;
-	else                   collectedPixels[cluster.key()]=true;
+	else if (cluster.isPixel()) collectedPixels[cluster.key()]=true;
```

Comment:
Right now, I have introduced `isPixel`, `isStrip` and `isPhase2` for clusters. In the future probably is going to be split also isPhase2 - we will see what we need and where.